packages/indexer-service/src/server/__tests__/server.test.ts

```diff
@@ -64,18 +64,14 @@ const setup = async () => {
   contracts = await connectContracts(getTestProvider('sepolia'), 11155111, undefined)
   sequelize = await sequelize.sync({ force: true })
   const statusEndpoint = 'http://127.0.0.1:8030/graphql'
-  indexingStatusResolver = new IndexingStatusResolver({
-    logger: logger,
-    statusEndpoint,
-  })
+  const queryEndpoint = 'http://127.0.0.1:8000/'
```

Comment:
I think this got clobbered when we were doing the rebase the other day @dwerner. Taking the opportunity to fixup here.